openhcl/underhill_core/src/worker.rs

```diff
@@ -2588,14 +2588,35 @@ async fn new_underhill_vm(
             .unwrap_or(!controllers.mana.is_empty());
         tracing::info!(enable_mnf, "Underhill MNF enabled?");
 
+        let max_version = env_cfg
+            .vmbus_max_version
+            .map(vmbus_core::MaxVersionInfo::new)
+            .or_else(|| {
```

Comment:
`map_or_else` exists

Reply:
You can't return `None` from the "else" function in `map_or_else`.

Reply:
You can't return `None` when using `map_or_else`.

Reply:
Ah, right you are.